.github/workflows/modules/Steps.pkl

```diff
@@ -0,0 +1,40 @@
+import "package://pkg.pkl-lang.org/github.com/stefma/pkl-gha/com.github.action@0.0.3#/GitHubAction.pkl"
+
+function checkout(customWith: Mapping<String, String|Number|Boolean>?): GitHubAction.Step = new {
+  name = "Checkout"
+  uses = "actions/checkout@v4"
+  with = customWith
+}
+
+function installPkl(): GitHubAction.Step = new {
+  name = "Install pkl"
+  uses = "pkl-community/setup-pkl@v0"
+  with {
+    ["pkl-version"] = "0.26.3"
+  }
+}
```

Comment:
There's no need to use a function for something like this in Pkl, you can just do:
```suggestion
installPkl: GitHubAction.Step = new {
  name = "Install pkl"
  uses = "pkl-community/setup-pkl@v0"
  with {
    ["pkl-version"] = "0.26.3"
  }
}
```

Reply:
Done here https://github.com/pkl-community/setup-pkl/pull/47/commits/e3b40c36d8f4f654ce498dd774c616c01fe4dc29